README.md

```diff
@@ -56,6 +56,9 @@ player.play();
 </script>
 ```
 
+## Documentation: API and Configuration
+PlayKit JS can be configured and controlled easily, click [here](docs/api.md) for details.
```

Comment:
PlayKit JS can be configured and controlled easily using the Player API. Click [here](docs/api.md) to learn more.

Also, why are we still using the word PlayKit? I think we need to stop using this terminology. Verify with Karin please.

Reply:
What do u suggest to use instead?
PlaykitJS refers to the library name